### PR TITLE
Log every 1000 items when exporting

### PR DESCRIPTION
**Pull request by meain**
---

#### Does this PR need a docs update or release note?

- [ ] :white_check_mark: Yes, it's included
- [ ] :clock1: Yes, but in a later PR
- [x] :no_entry: No

#### Type of change

- [ ] :sunflower: Feature
- [ ] :bug: Bugfix
- [ ] :world_map: Documentation
- [x] :robot: Supportability/Tests
- [ ] :computer: CI/Deployment
- [ ] :broom: Tech Debt/Cleanup

#### Issue(s)

* #<issue>

#### Test Plan

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: src/internal/archive/zip.go [hunk 1]
```diff
@@ -56,12 +57,22 @@ func ZipExportCollection(
...
 		for _, ec := range expCollections {
 			folder := ec.BasePath()
 			items := ec.Items(ctx)
 
 			for item := range items {
+				counted++
+
+				// Log every 1000 items that are processed
+				if counted%1000 == 0 {
+					log.Infow("zipping export items", "count_items", counted)
```
Comment (ryanfkeepers) [inline src/internal/archive/zip.go:73]:
```suggestion
					log.Infow("progress zipping export items", "count_items", counted)
```

File: src/pkg/export/consume.go [hunk 2]
```diff
@@ -29,6 +34,13 @@ func ConsumeExportCollections(
 		ictx := clues.Add(ctx, "dir_name", folder)
 
 		for item := range col.Items(ictx) {
+			counted++
+
+			// Log every 1000 items that are processed
+			if counted%1000 == 0 {
+				log.Infow("writing export items", "count_items", counted)
```
Comment (ryanfkeepers) [inline src/pkg/export/consume.go:41]:
```suggestion
				log.Infow("progress writing export items", "count_items", counted)
```

File: src/pkg/export/consume.go [hunk 3]
```diff
@@ -42,6 +54,8 @@ func ConsumeExportCollections(
 		}
 	}
 
+	log.Infow("wrote export items", "count_items", counted)
```
Comment (ryanfkeepers) [inline src/pkg/export/consume.go:57]:
```suggestion
	log.Infow("completed writing export items", "count_items", counted)
```

File: src/internal/archive/zip.go [hunk 4]
```diff
@@ -90,6 +101,8 @@ func ZipExportCollection(
 				}
 			}
 		}
+
+		log.Infow("zipped export items", "count_items", counted)
```
Comment (ryanfkeepers) [inline src/internal/archive/zip.go:105]:
```suggestion
		log.Infow("completed zipping export items", "count_items", counted)
```

**Review by ryanfkeepers — APPROVED** [review]

**Review by zackrossman — APPROVED** [review]
+1 to Keepers' suggestions

**Pull request merged**